demo-notebooks/guided-demos/0_basic_ray.ipynb

```diff
@@ -53,24 +53,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
```

Comment:
Do we care about these remaining `null`? Should be solveable by just restarting the notebook before committing changes?

Reply:
We should keep them null just for the sake of consistency :+1: